protobuf/v1/pool.proto

```diff
@@ -64,6 +67,7 @@ message Pool {
   uint64 used = 6;            // used bytes from the pool
   PoolType pooltype = 7;      // type of the pool
   uint64 committed = 8;       // committed size of all pool replicas (sum of capacities of all replicas)
+  uint32 cluster_size = 9;    // blobstor cluster size set during pool creation
```

Comment:
Please describe the unit of the size ("bytes").

Reply:
Done the changes.